spec/ParseAPI.spec.js

```diff
@@ -1267,6 +1267,35 @@ describe('miscellaneous', function () {
     });
   });
 
+  it('test cloud function query parameters with array of pointers', done => {
```

Comment:
We don't use chained promises anymore; why is this test not the same as the original fix for PS7?

Reply:
hmm. i copied the one in the fix. maybe i copied an older one? let me double check.

Reply:
I got it from here

https://github.com/parse-community/parse-server/pull/8784/files

Reply:
Yes, the test is different, no?

Reply:
Replaced incorrect test in https://github.com/parse-community/parse-server/pull/9012/commits/0cdf3f30c40cc80dba383fa14bd30a5fdcba4f29